aws/fake/cf.go

```diff
@@ -52,7 +62,10 @@ func (m *CFClient) DescribeStacks(in *cloudformation.DescribeStacksInput) (*clou
 }
 
 func (m *CFClient) CreateStack(params *cloudformation.CreateStackInput) (*cloudformation.CreateStackOutput, error) {
-	m.lastGeneratedTemplate = *params.TemplateBody
+	m.lastStackTags = params.Tags
+	m.lastStackParams = params.Parameters
+	m.lastStackTemplate = *params.TemplateBody
```

Comment:
Could params.TemplateBody be nil? Maybe its safer to use pointer as well.

Reply:
Ah, I see it was dereferenced in the existing code.

Reply:
It should not be possible, if this is the case this should be an error.